### PR TITLE
release note for 7.1.0

### PR DESCRIPTION
**Pull request by atsju**
I think we are due for 7.1.0
Feel free to update the release note in this branch

### CODE REVIEW

**Comment by gr5** [comment]
I agree!  I was thinking the same thing.  I'll look at the release notes and make comments.

File: RevisionHistory.html [hunk 1]
```diff
@@ -368,3 +368,25 @@
...
+    <li>Displayed waveFront is clearly identified</li>
+    <li>F2 shortcut can be used to rename waveFront</li>
+    <li>delete shortcut can be used to delete waveFront</li>
+    <li>enter shortcut can be used to dispaly selected waveFront</li>
+    <li>Right click gives access to a menu for all previous shortcuts</li>
+</ul>
+<li>Fix issue where regions sometimes fill in with data off by many wavelengths</li>
+<li>Several secondary windows are now closing automatically on main window close (but not all)</li>
+<li>Fix issue where Foucault view display "TextLabel" instead of number</li>
+<li>Fix regression in introduced in 7.0.0 not displaying effects of central obstruction in most displays</li>
```
Comment (gr5) [inline RevisionHistory.html:387]:
```suggestion
<li>Fix regression in introduced 7.0.0 not displaying effects of central obstruction in most displays</li>
```

File: RevisionHistory.html [hunk 2]
```diff
@@ -368,3 +368,25 @@
...
+<li>Remove unused interpolation drop down from rotate wavefront dialog</li>
+<li>Linux is not sensitive to jpg or other extension case anymore for selecting files</li>
+<li>Fix regression in introduced in 7.0.0 where regions might not be updated correctly when processing several igram</li>
```
Comment (gr5) [inline RevisionHistory.html:390]:
```suggestion
<li>Fix regression introduced in 7.0.0 where regions might not be updated correctly when processing several igram</li>
```

File: RevisionHistory.html [hunk 3]
```diff
@@ -368,3 +368,25 @@
...
 </ul></ul>
 </ul>
+
+<ul><li>Version 7.1.0</li>
+<ul>
+<li>Update project dependencies</li>
+<li>Fix regression introduced in 7.0.0 breaking "show only average" of astig stats</li>
+<li>Improved waveFront slection display</li>
+<ul>
+    <li>Displayed waveFront is clearly identified</li>
```
Comment (gr5) [inline RevisionHistory.html:378]:
```suggestion
    <li>Displayed waveFront is clearly identified with a green circle</li>
```

**Comment by gr5** [comment]
Please mention that you added logging:
Added logging to assist with debugging, particularly if there is a crash.  To view log file go to exe folder then DFTFringeLogs/log.txt

Also verb tense "fix", "remove" is not ideal.  Present or past tense better: "fixes" or "fixed".  "Removes"

Other than that it looks great.  Make these changes and I'll do a new release.

**Comment by atsju** [comment]
> Please mention that you added logging:

forgot this one. I though it was already in 7.0.0

Thanks for review

File: RevisionHistory.html [hunk 4]
```diff
@@ -371,22 +371,23 @@
...
 <ul>
-<li>Update project dependencies</li>
-<li>Fix regression introduced in 7.0.0 breaking "show only average" of astig stats</li>
+<li>Added logging to assist with debugging, particularly if there is a crash. To view log file go to exe folder then DFTFringeLogs/log.txt</li>
+<li>Updated project dependencies</li>
+<li>Fixed regression introduced in 7.0.0 breaking "show only average" of astig stats</li>
 <li>Improved waveFront slection display</li>
 <ul>
-    <li>Displayed waveFront is clearly identified</li>
+    <li>Displayed waveFront is clearly identifiedwith a green circle</li>
```
Comment (gr5) [inline RevisionHistory.html:379]:
```suggestion
    <li>Displayed waveFront is clearly identified with a green circle</li>
```

**Review by gr5 — APPROVED** [review]

**Pull request merged**